NuRadioReco/modules/RNO_G/channelGlitchDetector.py

```diff
@@ -114,3 +135,8 @@ def unscramble(trace):
             
             ch.set_parameter(chp.glitch, glitch_disc)
             ch.set_parameter(chp.glitch_test_statistic, glitch_ts)
+
+            # update glitching statistics
+            if not ch_id in self.events_glitching_per_channel:
+                self.events_glitching_per_channel[ch_id] = 0
+            self.events_glitching_per_channel[ch_id] += glitch_disc
```

Comment:
You can use `defaultdict(int)` to simplify the code to simply one line: `self.events_glitching_per_channel[ch_id] += glitch_disc`